<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>menu</string>
  </property>
  <widget class="QLabel" name="label_5wins">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>471</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>85</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5wins</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>540</y>
     <width>275</width>
     <height>76</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_vsAI">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>275</width>
     <height>75</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VS AI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_createServer">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>236</width>
     <height>76</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_joinServer">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>275</width>
     <height>76</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Join Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_local2Player">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>275</width>
     <height>76</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Local 2 Player</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_serverIp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
